docs/_docs/collections.md

```diff
@@ -51,7 +51,7 @@ defaults:
 
 Create a corresponding folder (for example, `<source>/_my_collection`) and add
 documents. YAML Front Matter is read in as data if it exists, and everything
-after it is available in the document's `content` attribute. If no YAML Front
+after it is accessible via the document's `content` attribute. If no YAML Front
```

Comment:
I only repeated it to avoid the ambiguity. Feel free to change this sentence to make it read how you think it sounds best. I'm actually heading out of town for a few days, so my communication will be spotty. (I didn't want to bring up capitalization issues with Front Matter versus front matter, or document versus Document, but I'll try to include something in a style guide at some future point so we maintain consistency.)